src/OpenSearch.Net.Auth.AwsSigV4/AwsSigV4HttpClientHandler.cs

```diff
@@ -20,19 +20,21 @@ internal class AwsSigV4HttpClientHandler : DelegatingHandler
 	{
 		private readonly AWSCredentials _credentials;
 		private readonly RegionEndpoint _region;
+		private readonly string _serviceId;
```

Comment:
According to https://docs.aws.amazon.com/general/latest/gr/signing-elements.html this should be called "service code". I see other clients use "service" or "service name". At least I think we shouldn't introduce a new one 😅